### DISCUSSION TITLE
67374bd918af07e2a97a1e34

### DISCUSSION THREAD

**Discussion by giscus** [Admin-ToxicGames/toxicgames#5] [General]
# 67374bd918af07e2a97a1e34

Discover a wide range of cracked games, mods, and resources at Toxic Games. Download for free and dive into endless gaming adventures!

https://toxicgames.in/67374bd918af07e2a97a1e34

**Comment by Admin-ToxicGames** [comment]
<h2>Tomb Raider Fix</h2>

If the game does not start after a while, then run the file " load " inside the application in the Tomb Raider .app/Contents/MacOS/

This works for any path, try this If your game is not working.

Instructions:
Go to the Tomb Raider.app/Contents/MacOS

and paste this code in the "load" file =>

```bash
#!/bin/bash
cd "$(dirname "$0")"
app_path="$(pwd)"
app_path=" $(find "${app_path%/*}/Game" -name "*.app" -depth 1)"
plist_path="${app_path}/Contents/Info.plist"
exec_name="$(grep -A1 CFBundleExecutable "$plist_path" | awk -F'>' '/string/{ print $2 }' |awk -F'<' '{ print $1 }')"
xattr -cr "${app_path}"
appID=$(cat "$(find "$(pwd)" -name 'steam_appid.txt')")
appID="$(cat steam_appid.txt)"
export SteamAppId=$appID
export SteamGameId=$appID
export STEAMEMU_SETTINGS_DIR="${app_path%/*}/../Config"
export DYLD_LIBRARY_PATH=$(echo $(pwd))
launchctl remove com.valvesoftware.steam .ipctool
pkill ipcserver
./reset >/dev/null 2>&1
./ipcserver >/dev/null 2>&1 &
ipcpid=$!
"${app_path}/Contents/MacOS/${exec_name}" #>/dev/null 2>&1
gamepid= $!
wait gamepid 2>/dev/null
kill $ipcpid >/dev/null 2>&1
wait $ipcpid 2>/dev/null
launchctl remove com.valvesoftware.steam.ipctool
pkill ipcserver
./reset >/dev/null 2>&1
./ipcserver >/dev/null 2>&1 &
ipcpid=$!
"${app_path}/Contents/MacOS/${exec_name}" #>/dev/null 2>&1
gamepid=$!
wait gamepid 2>/dev/null
kill $ipcpid >/dev/null 2>&1
wait $ipcpid 2>/dev/null
launchctl load -S Background /Users/$USER/Library/Application\ Support/Steam/com.valvesoftware .steam.ipctool.plist > /dev/null 2>&1

**Comment by misam199696** [comment]
“Tomb Raider” is damaged and can’t be opened. You should move it to the Trash. 
getting this while opening the game didn't run

**Reply by Admin-ToxicGames** [reply]
Check "Fix Installation Error" beside description.

**Comment by BlackBeastAK** [comment]
How to change the game audio language ?

**Comment by jikmlik** [comment]
how can i change the language of tomb raider in french ?

**Comment by jikmlik** [comment]
I connected my PS4 controller to play Tomb Raider, GTA 5, and Watch Dogs, but it all displays the Xbox controller on the Mac, but the buttons work as usual, but the screen displays the Xbox buttons.
